tests/monotouch-test/ImageIO/CGImageAnimationTest.cs

```diff
@@ -0,0 +1,186 @@
+﻿//
+// Unit tests for CGImageAnimation
+//
+// Authors:
+//	Whitney Schmidt <whschm@microsoft.com>
+//
+// Copyright 2020 Microsoft Corp. All rights reserved.
+//
+using System;
+using System.IO;
+#if XAMCORE_2_0
+using Foundation;
+using System.Threading.Tasks;
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
+using CoreGraphics;
+using ImageIO;
+#else
+using MonoTouch.CoreGraphics;
+using MonoTouch.Foundation;
+using MonoTouch.ImageIO;
+using MonoTouch.UIKit;
+#endif
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.ImageIO {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class CGImageAnimationTest {
+
+		NSUrl imageUrl;
+		NSData imageData;
+		int testValue;
+		CGImageAnimation imageAnimation;
+
+		void MyBlockSetValueZero (nint index, CGImage image, out bool stop)
+		{
+			stop = true;
+			testValue = 0;
+		}
+
+		void MyBlockSetValueOne (nint index, CGImage image, out bool stop)
+		{
+			stop = true;
+			testValue = 1;
+		}
+
+		[TestFixtureSetUp]
+		public void Init ()
+		{
+			TestRuntime.AssertXcodeVersion (11, 4);
+
+			imageUrl = NSBundle.MainBundle.GetUrlForResource ("hack", "gif"); // why not just initialize in declaration?
```

Comment:
Unless you need this one specifically let's reuse some existing tests resources. That minimize the .app size (so it builds and deploy faster).

e.g. `NSBundle.MainBundle.GetUrlForResource ("xamarin2", "png")`

Reply:
AFAIK, needs to be a gif and not a png :/

Reply:
AFAIK, needs to be a gif and not a png :/

Reply:
It needs to be a .gif for animation :/